src/ipaperftest/core/main.py

```diff
@@ -166,6 +166,16 @@ def run(self, ctx):
     help="Time range in minutes to spread auths in AuthenticationTest",
     default=0,
 )
+@click.option(
+    "--expected-result-type",
+    help="Type of expected result.",
+    type=click.Choice(["time", "time_unit", "no_errors"]), default="no_errors"
+)
+@click.option(
+    "--expected-result",
+    help="Expected result of the test",
+    type=click.INT
+)
```

Comment:
I think we'll need to clarify this in some way. This is mostly used for time now, which I assume means it's comparing the expected time vs the actual time, but I wonder if it would be used for something else in the future. e.g if it's "time" then maybe state that the unit is seconds, for example. Maybe README.md is sufficient.

Reply:
I added a section in README.